docs/pnp/migrate-vsto-to-office-web-add-in-shared-code-library-tutorial.md

```diff
@@ -0,0 +1,427 @@
+
+## Shared code library
+
+This guide will walk you through the steps of identifying and sharing common code between your VSTO Add-in and a modern Office web add-in. It uses a very simple VSTO Add-in example for the steps so that you can focus on the skills and techniques you will need for working with your own VSTO Add-ins.
+
+The following diagram shows how the shared code library works for migration. Common code is refactored into a new shared code library. The code can remain written in its original language, such as C# or VB. This means you can continue using the code in the existing VSTO Add-in by creating a project reference. When you create the Office web add-in, it will also use the shared code library by calling into it through REST APIs.
+
+![Diagram of VSTO Add-in and Office web add-in using a shared code library](../images/vsto-migration-shared-code-library.png)
+
+Skills and techniques in this guide:
+
+- Create a shared class library by refactoring code into a .NET class library.
+- Create a REST API wrapper using ASP.NET Core for the shared class library.
+- Call the REST API from the Office web add-in to access shared code.
+
+## Prerequisites
+
+To set up your development environment:
+
+1. Install [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/).
+2. Install the following workloads:
+    a. ASP.NET and web development
+    b. .NET Core cross-platform development
+    c. Office/SharePoint development
+    d. Visual Studio Tools for Office (VSTO) Note that this is an Individual component.
+
+You will also need the following:
+
+- An Office 365 account. You can join the [Office 365 Developer Program](https://aka.ms/devprogramsignup) that includes a free 1 year subscription to Office 365.
+- A Microsoft Azure Tenant. This add-in requires Azure Active Directiory (AD). Azure AD provides identity services that applications use for authentication and authorization. A trial subscription can be acquired here: [Microsoft Azure](https://account.windowsazure.com/SignUp).
+
+## The Cell analyzer VSTO Add-in
+
+This guide uses the [VSTO Add-in shared library for Office web add-in](https://github.com/OfficeDev/PnP-OfficeAddins/tree/vstoshared/Samples/VSTO-shared-code-start) PnP solution. The **/start** folder contains the VSTO Add-in solution that you will migrate. Your goal is to migrate the VSTO Add-in to a modern Office web add-in by sharing code when possible.
+
+> [!NOTE]
+> The sample uses C# but you can apply the techniques in this guide to a VSTO Add-in written in any .NET language.
+
+1. Download the [VSTO Add-in shared library for Office web add-in](https://github.com/OfficeDev/PnP-OfficeAddins/tree/vstoshared/Samples/VSTO-shared-code-start) PnP solution to a working folder on your computer.
+2. Start Visual Studio 2019 and open the **/start/Cell-Analyzer.sln** solution.
+3. From the **Debug** menu choose **Start Debugging**.
```

Comment:
```suggestion
3. On the **Debug** menu, choose **Start Debugging**.
```
As per: https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

You could also say  "Go to the **Debug** menu and select **Start Debugging**.